src/mad_spark_alt/core/qadi_prompts.py

```
@@ -57,25 +57,25 @@ def get_abduction_prompt(user_input: str, core_question: str, num_hypotheses: in
 
-H3: [Third approach title]
+3. [Third approach title]
 [Detailed explanation of the third approach with specific steps, technologies, methodologies, and implementation details. This should be a comprehensive paragraph explaining how this approach works, what resources it requires, and why it addresses the core question effectively.]""" + (f"""
 
-H4: [Fourth approach title]
+4. [Fourth approach title]
 [Detailed explanation of the fourth approach with specific steps, technologies, methodologies, and implementation details. This should be a comprehensive paragraph explaining how this approach works, what resources it requires, and why it addresses the core question effectively.]
 
-H5: [Fifth approach title]
+5. [Fifth approach title]
 [Detailed explanation of the fifth approach with specific steps, technologies, methodologies, and implementation details. This should be a comprehensive paragraph explaining how this approach works, what resources it requires, and why it addresses the core question effectively.]""" if num_hypotheses > 3 else "") + (f"""
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)